presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -137,4 +140,35 @@ public GlueHiveMetastoreConfig setCatalogId(String catalogId)
         this.catalogId = Optional.ofNullable(catalogId);
         return this;
     }
+
+    public boolean isUseInstanceCredentials()
+    {
+        return useInstanceCredentials;
+    }
+
+    @Config("hive.metastore.glue.use-instance-credentials")
+    public GlueHiveMetastoreConfig setUseInstanceCredentials(boolean useInstanceCredentials)
+    {
+        this.useInstanceCredentials = useInstanceCredentials;
+        return this;
+    }
+
+    public Optional<String> getAwsCredentialsProvider()
+    {
+        return awsCredentialsProvider;
+    }
+
+    @Config("hive.metastore.glue.aws-credentials-provider")
+    public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsProvider)
+    {
+        this.awsCredentialsProvider = Optional.ofNullable(awsCredentialsProvider);
+        return this;
+    }
+
+    @AssertTrue
+    public boolean validateUniqueCredentialProvider()
+    {
+        // if both credential provider not set, default credential chain will be used
+        return !useInstanceCredentials || !awsCredentialsProvider.isPresent();
```

Comment:
base credentials can also be set.
It should be either:
1. no base credentials, no `useInstanceCredentials` and no `awsCredentialsProvider` is set
2. At most one of: base credentials, `useInstanceCredentials` and `awsCredentialsProvider` is set

Reply:
oh, I see, credential in confg !

Reply:
should we set useInstanceCredentials default value as false ? @sopel39